Update LastPass and CloudFlare WARP references (#33)
parallel-markets/handbook: security/devices.md

## Network :satellite:
In general, we operate with a stance of [Zero Trust](https://www.cloudflare.com/learning/security/glossary/what-is-zero-trust/).  We assume that our internal networks could be compromised and seek to secure resources rather than networks.  Given the nature of our [remote workforce]({{ site.baseurl }}{% link work/index.md %}) and our utilization of cloud services, we cannot assume that resources are located within an enterprise-owned network boundary.

> To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client (downloads can be found [here](https://developers.cloudflare.com/warp-client/get-started/)).

Reviewer: Most people won't be able to install directly due to local permissions (though they can use the self service app)

```suggestion
To help reduce networking risks, we utilize [Cloudflare for Teams](https://www.cloudflare.com/teams/).  Everyone should install and utilize the Warp client via the Self Service app.
```

Author: Good call. Didn't see it in Self Service initially, but I do now.
